### ISSUE TITLE
st.popover examples erroring out

### ISSUE THREAD

**Issue by digitalghost-dev** [streamlit/docs#996]
### Summary

When visiting the `st.popover` [docs](https://docs.streamlit.io/library/api-reference/layout/st.popover), the examples are erroring out and not providing an example

### Steps to reproduce

Code snippet:
1. Visit the `st.popover` [docs](https://docs.streamlit.io/library/api-reference/layout/st.popover) and scroll down to the examples.

Screenshot(s):

![image](https://github.com/streamlit/docs/assets/86637723/dde5caad-dbe9-410d-b082-0a925bc1ed78)

**Comment by snehankekre** [comment]
Hi @digitalghost-dev 👋  thanks for reporting this issue.

We've discovered an inefficient use of CPU for Streamlit version 1.32.0 and 1.32.1. Apps deployed on Community Cloud are temporarily capped at version 1.31.1. Patch 1.32.2 has been released on PyPi and is available for development. The limitation for Community Cloud is expected to be lifted on Monday, March 18.

As st.popover is available in versions >=1.32.0, the app throws an exception since it is using version 1.31.1. This issue should be resolved ideally by EOD.
